deb/build/debian/control

```diff
@@ -11,6 +11,4 @@ Architecture: all
 Depends: ${misc:Depends}, daemon, adduser, procps, psmisc, net-tools, default-jre-headless (>= 2:1.7) | java7-runtime-headless
 Conflicts: hudson
 Replaces: hudson
-Description: continuous integration system
- Jenkins is an application that monitors executions of repeated jobs,
- such as building a software project or jobs run by cron.
+Description: @@DESCRIPTION_FILE_DEB@@
```

Comment:
Alternately, you can define this environment variable within your build.sh script (assuming this is all done bash-side and doesn't go through the makefile madness it should work to read contents to/from variables).